src/openrpc.json

```diff
@@ -0,0 +1,53 @@
+{
+  "openrpc": "1.0.0",
+  "info": {
+    "title": "foo",
+    "version": "1.0.0"
```

Comment:
ideally this should be in lockstep with the package.json version.

need to make a github action alternative to https://github.com/open-rpc/semantic-release-plugin to bump this

Reply:
but this interface is the transport plugin interface so it wont change. but we have other openrpc document based repositories that need to update their `info.version` in sync